synchronizer/models.py

```diff
@@ -395,6 +395,16 @@ class Connector(db.Model):
         backref='connector_types'
     )
 
+    @property
+    def is_valid(self):
+        connector = ConnectorManager.get_connector(self.connector_type.name)
+        return connector.validate_connector(
+            server=self.server,
+            login=self.login,
+            api_token=self.api_token,
+            connector_type=self.connector_type.name,
```

Comment:
Do not need to pass `connector_type` into `validate_connector` method. By the way, we can rename method to `validate` to avoid duplication of text (like connector.validate_connector).